src/explorer/mod.rs

```diff
@@ -32,6 +32,8 @@ pub struct GitExplorer {
     stop_condition_i: usize,
     stop_conditions: Vec<Option<BranchData>>,
     nodes_len: usize,
+    abort: bool,
```

Comment:
abort and limit_stack attributes created on GitExplorer struct, the idea is to remove the arguments [here](https://github.com/KarlHeitmann/git_explorer/pull/3/files#diff-94650c3b1172d7a5565e9239f9dd69f75e342eed9dd53ed6ed8f00321b1e76f2L211-L212). Separate responsibilities, they are not necessary to be as arguments